### PR TITLE
[plugin.video.catchuptvandmore] 0.1.3

### PR DESCRIPTION
**Pull request by SylvainCecchetto**
### Description
* Add new channels (Catch up TV and live TV)
* Add download video functionality
### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.catchuptvandmore/resources/lib/channels/fr/6play.py [hunk 1]
```diff
@@ -203,15 +259,15 @@ def list_shows(params):
...
                 'plot': params.program_desc
             }
         }
         shows.append({
-            'label': common.addon.get_localized_string(30101),
+            'label': common.ADDON.get_localized_string(30101),
             'thumb': params.program_img,
-            'fanart': params.program_fanart,
-            'url': common.plugin.get_url(
+            'fanart': program_fanart,
```
[the remark this answers is not in the document]
Comment (SylvainCecchetto) [reply plugin.video.catchuptvandmore/resources/lib/channels/fr/6play.py]: Resolved

File: plugin.video.catchuptvandmore/addon.xml [hunk 2]
```diff
@@ -1,12 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.catchuptvandmore"
        name="Catch-up TV &amp; More"
-       version="0.1.2"
+       version="0.1.3"
        provider-name="SylvainCecchetto">
   <requires>
-    <import addon="xbmc.python" version="2.25.0"/>
+    <!-- TODO before official repo submission <import addon="xbmc.python" version="2.25.0"/> -->
```
Comment (Lunatixz) [inline plugin.video.catchuptvandmore/addon.xml]: = ) Please correct

File: plugin.video.catchuptvandmore/addon.xml [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+    <import addon="xbmc.python" version="2.24.0"/>
     <import addon="script.module.beautifulsoup4" version="4.3.1"/>
     <import addon="script.module.requests" version="2.9.1"/>
+    <import addon="script.module.youtube.dl" version="17.518.2"/>
+    <import addon="script.module.elementtree" version="1.2.8"/>
```
Comment (Lunatixz) [inline plugin.video.catchuptvandmore/addon.xml]: elementtree is already part of python, no need for extra module.

**Comment by SylvainCecchetto** [comment]
Thanks for the review. I just modified the add-on.xml file.

**Comment by SylvainCecchetto** [comment]
Hi!
Is there problem with the add-on? Or maybe it's long to review?
Would you like for the next time that we propose a pull request earlier so that there is less change to review in the code?
Thanks a lot!

**Comment by Razzeee** [comment]
Hey, 
this has been quiet a task to review. If you can, send updates more frequently in the future. But not too frequently ;)

Otherwise LGTM!

**Pull request merged**